docs/getting-started/how-to/troubleshoot-common-codespaces-issues/index.md

```diff
@@ -47,3 +47,28 @@ However, you will find that any changes you make will not be preserved when you
 If you wish to make changes that will apply whenever anyone opens your project, then consider using a [Workspace Settings](https://code.visualstudio.com/docs/getstarted/settings#_workspace-settings) file in your project.
 
 If you wish your settings and extensions to be available anywhere you open VS Code (including in Codespaces), then consider using [Settings Sync](https://code.visualstudio.com/docs/editor/settings-sync).
+
+## Previewing HTML files doesn't work
+
+If you are using a codespace to author an HTML file, for example a report, you may wish to see a live preview of this and for this purpose many will turn to the [Live Preview](https://marketplace.visualstudio.com/items?itemName=ms-vscode.live-server) VS Code extension from Microsoft.
+However, if you install and try to use this extension in a codespace, you might experiences errors such as _"This content is blocked. Contact the site owner to fix the issue."_ or the preview window appearing blank.
+This is a [known issue](https://github.com/microsoft/vscode-livepreview/issues/111) which is currently awaiting a fix from the extension's developers.
+
+### Workaround
```

Comment:
question: I guess this so, but does this work without doing the [dance described](https://github.com/microsoft/vscode-livepreview/issues/111#issuecomment-1095606237) in this post?

I know when I tried to get Live Preview working a month or two ago it was a bit of a hassle, and I'm not sure what steps I took to get it working.